src/app-account.js

```diff
@@ -29,7 +29,10 @@ app.get('/needs-bootstrap', (req, res) => {
 
   res.send({
     status: 'ok',
-    data: { bootstrapped: rows.length > 0 }
+    data: {
+      isActual: true,
```

Comment:
Do we really need this constant? Wouldn't a 200 response from this endpoint be sufficient to tell if it's an actual server?

Reply:
Sure.